src/libraries/Microsoft.Bcl.AsyncInterfaces/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
It'd be good to clean up the suppressions in this diff.  Not sure why you're hitting these, but if you need some help wrangling the dotnet/runtime infra reach out to Steve to get some tips for working on these libraries and adding API.